lib/srv/discovery/access_graph_test.go

```diff
@@ -136,6 +135,43 @@ func TestServer_updateDiscoveryConfigStatus(t *testing.T) {
 					{
 						State:               "DISCOVERY_CONFIG_STATE_SYNCING",
 						ErrorMessage:        nil,
+						DiscoveredResources: 0,
+						LastSyncTime:        clock.Now(),
+					},
+				},
+			},
+		},
+		{
+			name: "test multiple aws sync fetchers",
```

Comment:
Could we test error merging too?

Reply:
Adding more detail: error merging and mixed success/failures statuses, so we know it's keeping the "worst" status as the final one.

Reply:
Tests added